src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -368,18 +368,38 @@ def responsive_namespace
 
   def sign_up_link(css_class: nil)
     return unless can_sign_up?
+
     if proxy_mode?
-      link_to('Sign Up', sign_up_params[:url], class: css_class)
+      link_to(sign_up_params[:url], class: css_class) do
+        link_content('Sign Up', css_class, 'fa-signature')
```

Comment:
Hm, this is the first time I have seen this icon in this context. Not sure this helps. How about `user-plus`?

![Screenshot from 2020-03-06 11-36-44](https://user-images.githubusercontent.com/514785/76076260-cc94d980-5f9e-11ea-9186-4687012c0b5e.png)

Reply:
In fact, that was my first idea... Let's see if we finally use icons for Log In and Sign up.

Reply:
I've already replaced the signature icon with `user-plus` one and updated the screenshots on the PR's description.